### DISCUSSION TITLE
Meus Plots nao aparecem

### DISCUSSION THREAD

**Discussion by marcostraderminer** [Chia-Network/chia-blockchain#7434] [Support]
Boa noite, 

Ja tenho 84 plots em uma maquina e 2 em outra, porem meus plots nao aparecem, ficam carregando e nao aparecem.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
